src/utils/selectors.ts

```diff
@@ -140,31 +139,17 @@ export const isUserPurchaseHistoryRequestedSelector: SparkleSelector<boolean> =
   "userPurchaseHistory"
 );
 
-const DAYS_AGO = getDaysAgoInSeconds(VENUE_CHAT_AGE_DAYS);
-const HIDE_BEFORE = roundToNearestHour(DAYS_AGO);
-
-export const unreadMessagesSelector = (state: RootState) => {
-  const user = authSelector(state);
-  const privateChats = privateChatsSelector(state) ?? [];
-
-  return privateChats.some(
-    (message) =>
-      message.from !== user?.uid &&
-      message.deleted !== true &&
-      message.type === "private" &&
-      message.ts_utc.seconds > HIDE_BEFORE &&
-      message.isRead === false
-  );
-};
```

Comment:
Nowhere. It was deleted since it didn't actually work in an intended way. We need unread messages from unique people. The logic was that is currently in privateChats hook

Reply:
Fair, makes sense.